src/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.cs

```diff
@@ -113,8 +113,6 @@ public static void PrepareMethod(RuntimeMethodHandle method, RuntimeTypeHandle[]
             }
         }
 
```

Comment:
This is public API. It is not ok to delete it: https://github.com/dotnet/corefx/blob/master/src/System.Runtime/ref/System.Runtime.cs#L6785

Reply:
Thanks for spotting it!